src/main/java/org/folio/des/service/FolioTenantService.java

```diff
@@ -67,4 +92,21 @@ protected void afterTenantDeletion(TenantAttributes tenantAttributes) {
     scheduledJobsRemover.deleteJobs(tenantId);
     oldJobDeleteScheduler.removeJobs(tenantId);
   }
+
+  protected boolean shouldUpdateBursar(TenantAttributes tenantAttributes) {
```

Comment:
do I understand correctly that it will return true for 3.2.0 3.2.1 3.2.2 etc version?

why not to incapsulate everything in  bursarMigrationService and call only bursarMigrationService.migrate() in FolioTenantService? bursarMigrationService.migrate() will check everything and call required migrations

Reply:
Sure, I will rearrange that.
and yes, you're correct; I'm assuming this would be released in a 4.0.0 for Q

Reply:
ah, ok, now I got it - missed that you use from version